modules/flowable-engine/src/test/java/org/flowable/engine/test/concurrency/ConcurrentEngineUsageTest.java

```diff
@@ -147,11 +147,9 @@ public void run() {
                     retryStartProcess(drivingUser);
                     finishTask = true;
 
-                    if (numberOfProcesses == 0) {
-                        // Make sure while-loop doesn't stop when processes are
-                        // all started
-                        tasksAvailable = taskService.createTaskQuery().taskAssignee(drivingUser).count() > 0;
-                    }
+                    // Make sure while-loop doesn't stop when processes are all started
```

Comment:
Hi @dbmalkovsky , I don't think this change is correct. Now we are always testing if tasks are available and before it was only done when numberOfProcesses == 0. Maybe this whole if clause can be removed because numberOfProcesses is always > 0 in this part of the code?